src/crypto/x509/x509.go

```diff
@@ -2023,7 +2023,7 @@ func CreateCertificateRequest(rand io.Reader, template *CertificateRequest, priv
 
 		b, err := asn1.Marshal(attr)
 		if err != nil {
-			return nil, errors.New("x509: failed to serialise extensions attribute: " + err.Error())
+			return nil, fmt.Errorf("x509: failed to serialise extensions attribute: %w", err)
```

Comment:
I had an urge to fix this existing type, but I did not.

```suggestion
			return nil, fmt.Errorf("x509: failed to serialize extensions attribute: %w", err)
```